motto/agents/agent.py

```diff
@@ -1,10 +1,16 @@
-import openai
+from openai import OpenAI
 
+client = OpenAI()
+
+class EchoResponse:
```

Comment:
Since this response class can be useful for other agents, I'd rename it to just `Response`